tests/server/test_server_status.py

```diff
@@ -52,6 +54,12 @@ async def test_server_status(server, client, agent, environment):
     assert "features" in status
     assert len(status["features"]) > 0
 
+    assert status["python_version"] == sys.version
+    postgresql_version = await postgresql_client.fetchval("SELECT version();")
+    # Assert that the output is `PostgreSQL X.Y[.Z] ...`
+    assert re.match(r"^(PostgreSQL \d+(?:\.\d+)+)", postgresql_version)
```

Comment:
I have the feeling that this assertion is too strict. It will fail if Postgres decides to change the format of this function. Time will tell.